### PR TITLE
Transpose of a Square Matrix using C++

### PR DESCRIPTION
**Pull request by somya1252**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)
#4867

### CODE REVIEW

File: C-Plus-Plus/other/Transpose_Of_3X3_Matrix.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/* Transpose of a matrix is a new matrix in which the rows
+of the original matrix are the columns now and vice versa.
+If the given matrix is 1 2 3
+                       4 5 6
+                       7 8 9
+
+then the transpose if  1 4 7
+                       2 5 8
+                       3 6 9
+*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/other/Transpose_Of_3X3_Matrix.cpp]: - Add Multi-Line Comments 
- Add a beginner-friendly description at the top

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Shift the logic to another function and use main for driver code

File: C-Plus-Plus/other/Transpose_Of_3X3_Matrix.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Transpose_Of_3X3_Matrix.cpp]: Don't push your primary code logic in the `main()` function

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@somya1252 Do the same for any square matrix

File: C-Plus-Plus/other/Transpose_Of_Square_Matrix.cpp [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        }
+    }
+}
+
+int main() {
+    // int n;
+    // cout<<"Enter the dimension of the matrix:";
+    // cin>>n;
+
+    int a[N][N];
```
Comment (siddharth25pandey) [inline C-Plus-Plus/other/Transpose_Of_Square_Matrix.cpp:24]: Input of N is missing

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@somya1252 You are not taking input of the size **N** then how you are taking input for the square matrix. I request you before submitting any PR. Please try to execute it.

**Review by siddharth25pandey — APPROVED** [review]
@somya1252 LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @somya1252 for her code

**Pull request merged**